### PR TITLE
Add ability to resend code

### PR DESCRIPTION
**Pull request by iDiogenes**
Added the ability for a user to resend the auth code from the two_factor_auth page.

### CODE REVIEW

**Comment by Houdini** [comment]
Thanks, could you please added some tests?

**Comment by iDiogenes** [comment]
Yes. I should have time to take care of this toward the end of the week.

File: app/controllers/devise/two_factor_authentication_controller.rb [hunk 1]
```diff
@@ -27,6 +27,11 @@ def update
     end
   end
 
+  def resend_code
+    resource.send_two_factor_authentication_code
+    redirect_to user_two_factor_authentication_path, notice: "Your authentication code has been sent."
```
Comment (monfresh) [inline app/controllers/devise/two_factor_authentication_controller.rb]: What do you think about localizing this notice, so that people can customize it?

**Comment by amandaporto** [comment]
Planning to merge this anytime soon?

**Comment by Houdini** [comment]
done

**Comment by monfresh** [comment]
@iDiogenes I know this has already been merged, but it would be great if you could please submit a new PR with tests for this. Thanks!

**Pull request merged**